pkg/controllers/management/drivers/kontainerdriver/kontainerdriver.go

```diff
@@ -349,36 +327,69 @@ func (l *Lifecycle) Updated(obj *v3.KontainerDriver) (runtime.Object, error) {
 		return obj, nil
 	}
 
-	if obj.Spec.BuiltIn {
+	if obj.Spec.BuiltIn && v3.KontainerDriverConditionActive.IsTrue(obj) {
```

Comment:
I don't think this check is appropriate for a couple reasons:
1) There doesn't appear to be a way for a builtin driver to get `Active` false
2) If there was then we would try to download a binary that doesn't exist and fail out.
I think this condition check should be removed.